flow/layer_snapshot_store.cc

```diff
@@ -10,20 +10,20 @@
 namespace flutter {
 
 LayerSnapshotData::LayerSnapshotData(int64_t layer_unique_id,
-                                     fml::TimeDelta duration,
-                                     sk_sp<SkData> snapshot)
+                                     const fml::TimeDelta& duration,
+                                     const sk_sp<SkData>& snapshot,
+                                     const SkRect& bounds)
     : layer_unique_id_(layer_unique_id),
       duration_(duration),
-      snapshot_(snapshot) {}
+      snapshot_(snapshot),
+      bounds_(bounds) {}
```

Comment:
is it also possible to send the original image size (the total size of the screen that the layer is a part of)? We'll need that in addition to the layer size and offset so that we can scale the outline box we'd like to draw in DevTools accordingly